python/res/enkf/queue_config.py

```diff
@@ -57,11 +57,11 @@ def __init__(self, user_config_file=None, config_content=None, config_dict=None)
 
         if config_dict is not None:
                 c_ptr = self._alloc_full(
-                                            config_dict[ConfigKeys.JOB_SCRIPT],
-                                            config_dict[ConfigKeys.USER_MODE],
-                                            config_dict[ConfigKeys.MAX_SUBMIT],
-                                            config_dict[ConfigKeys.NUM_CPU],
-                                            config_dict[ConfigKeys.QUEUE_SYSTEM]
+                                            config_dict.get(ConfigKeys.JOB_SCRIPT),
```

Comment:
None of these values can, or should be None for ERT to function

Reply:
Ok,  reverting it back